README.md

```diff
@@ -1,14 +1,28 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
 ## Latest Release: v2.12
+
 [Read the specification](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md)
 
 [Release Notes for past versions](release-notes.md)
 
-## Working Changes
+## SDLC
+
+### Working Changes
```

Comment:
sort of. yes it should be ready to go but I'm used to things being called "release candidate" when the group is specifically asking for it to be reviewed for the purposes of being the next release.  Might just be picky - if others are ok with it I'll survive  :-)